packages/fx-core/src/plugins/resource/function/utils/telemetry-helper.ts

```diff
@@ -75,4 +76,16 @@ export class TelemetryHelper {
       (e: SystemError | UserError) => this.sendErrorEvent(eventName, e, properties, measurements)
     );
   }
+
+  static sendFallbackEvent(
+    e: FunctionPluginError = new UnknownFallbackError(),
+    properties: { [key: string]: string } = {},
+    measurements: { [key: string]: number } = {}
+  ) {
+    this.fillCommonProperty(properties);
+    properties[TelemetryKey.ErrorMessage] = e.message;
+    properties[TelemetryKey.ErrorCode] = e.code;
+
+    this.ctx?.telemetryReporter?.sendTelemetryEvent(FunctionEvent.scaffoldFallback, properties, measurements);
```

Comment:
NIT: The event name doesn't match the method name, you send a specific `scaffoldFallback` event in a general `sendFallbackEvent` method.

Reply:
fixed